### ISSUE TITLE
Test Samba in PressBoxx from Windows

### ISSUE THREAD

**Issue by mikeschinkel** [pressboxx/console#14]
This is blocked by https://github.com/pressboxx/api/issues/12.

**Comment by clubdeuce** [comment]
SMB shares can be mounted in Mac OSX.

**Comment by mikeschinkel** [comment]
@clubdeuce This ticket was for @sarutole to test in Windows.  _(I should have explicitly stated "Windows.")_

**Comment by clubdeuce** [comment]
Then is this not a duplicate of pressboxx/app#8?
